.github/workflows/static-scan.yml

```diff
@@ -1,5 +1,11 @@
 name: Go-static-analysis
 on: [push, pull_request]
+paths-ignore:
+  - 'README.md'
+  - '**/README.md'
+  - 'backup/**'
+  - '**/*.draft.md'
+  - '.github/workflows/*.yml'
```

Comment:
I am not sure we need this
let's not modify this file for now, as it will conflict with #85

Reply:
also this break the action.